routers/web/repo/release.go

```diff
@@ -298,6 +299,14 @@ func SingleRelease(ctx *context.Context) {
 		release.Title = release.TagName
 	}
 
+	if ctx.IsSigned && !release.IsTag {
+		err = activities_model.SetReleaseReadBy(ctx, release.ID, ctx.Doer.ID)
+		if err != nil {
+			ctx.ServerError("SetReleaseReadBy", err)
```

Comment:
TBH I really had enough for such copy-paste `if ... err != nil { ctx.ServerError(...) }`. It's difficult to read, the message mismatches, and the "return" is easy to forget. 

Not really related to this PR, but why not introduce a better approach (I have proposed)

Reply:
Thanks for pointing out. It's a better method than this one. But maybe it should be another PR.

Reply:
The typo has been fixed in 5dc32a3